packages/teleterm/src/ui/ClusterConnect/ClusterLogin/FormLogin/FormLogin.tsx

```diff
@@ -170,12 +170,6 @@ export default function LoginForm(props: Props) {
                         mb={0}
                       />
                     )}
-                    {mfaType.value === 'u2f' && isProcessing && (
-                      <Text typography="body2" mb={1}>
-                        Insert your hardware key and press the button on the
-                        key.
-                      </Text>
-                    )}
```

Comment:
I think the Teleterm code might require some more changes since that's the only place that asks users to press the button on the hardware key, but with webauthn that is still required, isn't it?

I'll try to take a look in an hour but if I don't manage to then I'll submit a patch on Monday.

Reply:
the webauthn relies on the browsers dialog. for u2f, this prompt was needed b/c chrome didn't have a browser dialog (but firefox did)

Reply:
Thanks! I also didn't realize that in Teleterm there's a `PromptHardwareKey` component for that.

Reply:
@ravicious i just played with teleterm, and i now understood your concern, did not know how teleterm worked at all :sweat_smile: thanks for the check!